BlueShift-iOS-SDK/InApps/BlueShiftNotificationModalViewController.m

```diff
@@ -439,9 +439,71 @@ - (CGFloat)getActionButtonXPosition:(UIView *)parentView childWidth:(CGFloat)wid
     : xPadding;
 }
 
+- (CGSize)getAutoImageSizeForNotificationView {
+    float width = 0;
+    float height = 0;
+    
+    // Check if this modal is image modal
+    if ([self isBackgroundImagePresentForNotification:self.notification] && (self.notification.templateStyle.width < 0 || self.notification.templateStyle.height < 0)) {
```

Comment:
this means if both width and height are a positive number, we will not respect the image's aspect ratio. right?

Reply:
in android, I am treating the positive height and width as my maximum space available. ex; if the width is 80% I will not take more than 80%. But width can be less than 80% based on the image's aspect ratio.

Reply:
iOS implementation is little different, if the width height are in +ve then those values will be used irrespective of aspect ratio.